federatedscope/core/auxiliaries/data_builder.py

```diff
@@ -410,18 +422,46 @@ def load_huggingface_datasets_data(name, splits=None, config=None):
         assert 'max_len' in raw_args, "Miss key 'max_len' in " \
                                       "`config.data.args`."
         filtered_args = filter_dict(load_dataset, raw_args)
-        dataset = load_dataset(path=config.data.root,
-                               name=name,
-                               **filtered_args)
+        logger.info("Begin to load huggingface dataset")
+        if "hg_cache_dir" in raw_args:
+            hugging_face_path = raw_args["hg_cache_dir"]
+        else:
+            hugging_face_path = os.getcwd()
+
+        if "load_disk_dir" in raw_args:
```

Comment:
We could use a try here when `load_disk_dir` in `raw_args` but without the data file cached.